staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1191,6 +1191,14 @@ func TestList(t *testing.T) {
 		return nil, fields.Set{"metadata.name": pod.Name}, nil
 	}
 
+	oneLess := func(inputRv string) string {
```

Comment:
since what the "1" below was trying to do was find a resource version *before* any of the creates happened, I think it is clearer to do this:

```
	// grab the starting resource version before any creates
	initialList := &example.PodList{}
	if err := store.List(ctx, "/", storage.ListOptions{Predicate: storage.Everything}, initialList); err != nil {
		t.Fatalf("Unexpected List error: %v", err)
	}
	initialRV := initialList.ResourceVersion
```

before calling Create, and use `initialRV` instead of `"1"` or this oneLess approach

Reply:
ACK - yes, that also works! Sounds good